components/logged_in/index.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {autoUpdateTimezone} from 'mattermost-redux/actions/timezone';
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/channels';
+import {getLicense, getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getCurrentUser, shouldShowTermsOfService} from 'mattermost-redux/selectors/entities/users';
+import {GenericAction} from 'mattermost-redux/types/actions';
+import {Channel} from 'mattermost-redux/types/channels';
+
+import {browserHistory} from 'utils/browser_history';
+import {checkIfMFARequired} from 'utils/route';
+import {getChannelURL} from 'utils/utils';
+
+import {GlobalState} from 'types/store';
+
+import LoggedIn from './logged_in';
+
+export declare type GetStateFunc = () => GlobalState;
```

Comment:
This type already exists in `mattermost-redux/types/actions`, so it can be imported from there